Mod2/Labs/Week2/TDD.md

```diff
@@ -0,0 +1,41 @@
+## Lab Objectives
+- Practice class brainstorming
+- Use TDD to implement Library classes
+
+### Brainstorm Classes
+
+In small groups, review the requirements below.  Brainstorm the classes and methods that we would need to implement the functionality.
+
+```
+Library
+
+We need to be able to create a Library that has books and patrons.
```

Comment:
Along the same lines, I like the idea of having a library class that keeps track of all books and patrons. I think the instructions are fine as is, but maybe good to have an instructor note that we should come up with Library, Book, and Patron.